.storybook/themes/commerce.scss

```diff
@@ -1426,6 +1434,10 @@
 					components: (
 						link: (
 							font-size: var(--body-font-size),
+							filter: drop-shadow(0px 0px 0px rgba(0, 0, 0, 0)),
+						),
+						link-hover: (
+							color: var(--global-neutral-8),
```

Comment:
This also has unnecessary redundancy with these styles not needing to override the default breakpoint.